ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -225,3 +228,66 @@ def test_create_asset_with_address(publisher_ocean_instance):
     assert ocn.assets.create(
         asset.metadata, alice, [auth_service], data_token_address=token.address
     )
+
+
+def test_create_asset_with_owner_address(publisher_ocean_instance):
+    """Tests creation of the asset which has already an owner address."""
+    ocn = publisher_ocean_instance
+    alice = get_publisher_wallet()
+
+    sample_ddo_path = get_resource_path("ddo", "ddo_sa_sample.json")
+    asset = DDO(json_filename=sample_ddo_path)
+    asset.metadata["main"]["files"][0]["checksum"] = str(uuid.uuid4())
+    my_secret_store = "http://myownsecretstore.com"
+    auth_service = ServiceDescriptor.authorization_service_descriptor(my_secret_store)
+
+    token = ocn.create_data_token(
+        "DataToken1", "DT1", from_wallet=alice, blob="foo_blob"
+    )
+
+    assert ocn.assets.create(
```

Comment:
This assertion only checks whether the asset was created. My gut reaction is that more assertions are warranted. But after digging into `OceanAssets.create()` I realized that the `owner_address` parameter is silently ignored if `data_token_address` parameter is set, so this test is instruction-for-instruction identical to `test_create_asset_with_address`. In the end, I concluded that I don't know how to improve this test, but I wanted to write this comment anyway because I spent so much time on it.

Reply:
So, @DMats do you think it's useful to keep it or do you want to include it in `test_create_asset_with_address, because it's kind of similar? The difference consists in setting the parameters.

Reply:
If it were up to me, I'd remove the `owner_address` parameter from `OceanAssets.create()`. Then I'd remove this test (`test_create_asset_with_owner_address`). That said, I'd wait for confirmation from @calina-c or @trentmc to proceed.

Reasoning:

1. As far as I can tell, `owner_address` is not used in ocean.py or anywhere else in the ocean software stack. 
2. The [corresponding method in ocean.js](https://github.com/oceanprotocol/ocean.js/blob/main/src/ocean/Assets.ts#L69) does not have the `owner_address` parameter.

Caveats:

This would be an API-breaking change (assuming that OceanAssets is considered part of ocean.py's API).

Reply:
This will cause a breaking change of the interface if I will remove the `owner_address`. Been there, done that before.

Reply:
keep this, it was preciously discussed. `owner_address` stays :D

Reply:
will be addressed in #284